specification/sql/resource-manager/Microsoft.Sql/preview/2021-05-01-preview/examples/DistributedAvailabilityGroupsCreate.json

```diff
@@ -0,0 +1,46 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "testrg",
+    "managedInstanceName": "testcl",
+    "distributedAvailabilityGroupName": "dag",
+    "api-version": "2021-05-01",
```

Comment:
>2021-05-01 [](http://example.com/codeflow?start=20&length=10)

Make sure to update all your api-versions to: 2021-05-01-preview

Reply:
yea. @nemijato please also update it in the DSMainDev as well. DSMain is the upstream to auto-generate this swagger example files

Reply:
Done, tnx!